.github/workflows/lint.yaml

```diff
@@ -103,3 +103,18 @@ jobs:
         ignore_paths: 'vendor'
         disable_matcher: false
         format: gcc
+
```

Comment:
Can you please remove extra new line here

Reply:
done